constants.js

```diff
@@ -22,25 +23,16 @@ const checksWhitelist = {
     [labelEvent]: [],
     [synchronizeEvent]: [],
     [closeEvent]: [],
-    [editEvent]: []
+    [editEvent]: [],
```

Comment:
We should not have terminating `,` in an array list in JavaScript. Revert here and below.

Reply:
Ditto.